src/app/welcome-page/welcome-page.component.scss

```diff
@@ -237,11 +237,13 @@ h2 {
   }
   .hero-container img {
     height: 300px;
+    margin-bottom: -43px; // image has transparent bottom spacing
   }
 
   .description {
     align-items: flex-start;
     padding: 40px 0;
+    padding-bottom: 40px + 13;
```

Comment:
😆 ... It adds to whatever using is defined on 40 @kelvinkipruto . 40rem + 13 => 53rem, 40px + 13 => 53px. It's a SASS thing though: https://sass-lang.com/documentation/operators/numeric

Reply:
I need to really work on my Scss 🤯. Why not do units directly though? Like 53px 🤔?

Reply:
You usually do something like that to kind of self-explain what is happening @kelvinkipruto .
```
    padding: 40px 0;
    padding-bottom: 40px + 13;
```
Basically padding should be 40px but for the bottom we're adding 13 to it.. as step further could be explaining why e.g. because of negative margin or something.

If you just put `53px`, it's hard to know how one go there especially if that's not what's in the designs.

Reply:
aaah..., it's now clear.